package/tests/test_server.py

```diff
@@ -140,6 +145,19 @@ def teardown_function():
 
 @pytest.fixture
 def patched_get_project_context():
+    def get_project_context(
+        key: str = "context", **kwargs  # pylint: disable=bad-continuation
+    ):  # pylint: disable=unused-argument
+        mocked_context = mock.Mock()
+        mocked_context._get_pipeline = get_pipeline  # pylint: disable=protected-access
+        mocked_context.catalog = lambda x: None
+        mocked_context.pipeline = create_pipeline()
+        return {
+            "create_pipeline": create_pipeline,
+            "create_catalog": lambda x: None,
+            "context": mocked_context,
+        }[key]
+
     mock.patch("kedro_viz.server.get_project_context", new=get_project_context).start()
```

Comment:
```suggestion
    mocker.patch("kedro_viz.server.get_project_context", new=get_project_context)
```
Just flagging - ignore it here if you plan to address in a separate PR